frontend/packages/console-shared/src/components/badges/index.ts

```diff
@@ -1,2 +1,3 @@
+export * from './badge-factory';
 export { default as DevPreviewBadge } from './DevPreviewBadge';
 export { default as TechPreviewBadge } from './TechPreviewBadge';
```

Comment:
There are no direct references to `DevPreviewBadge` or `TechPreviewBadge` ... perhaps I remove them from direct export as in to angle usage through the `badge-factory`.

@christianvogt Thoughts? I could also add additional comments to the `getBadgeFromType` to suggest future users make use of model-driven properties. I'm not sure there is really anyway to force this though.

Reply:
I got no good arguments for or against. In which case I'd just go with the most flexible which is to keep the exports.